apps/schools/domains/ticket/components/currentTicket/index.tsx

```diff
@@ -0,0 +1,276 @@
+    return (
+        <div className={classnames(styles.commentBlock, is_internal_recipient && styles.privateComment)}>
+            <Image className={styles.image} src={is_sender ? family : employee} alt={'Sender icon'} />
+            <div className={styles.commentContainer}>
+                <div className={styles.titleComment}>
+                    <b>{is_sender ? `Семья ${familyName || 'не определена'}` : sender || 'Отправитель не определен'}</b>
+                    <span>{created_at ? format(new Date(created_at), 'dd.MM.yyyy HH:mm') : 'Время не определено'}</span>
+                    {is_internal_recipient && <Image src={lock} alt={'Lock icon'} />}
+                    <span>{is_internal_recipient ? 'Внутренний комментарий' : null}</span>
+                </div>
+                <div className={styles.comment}>{comment}</div>
+            </div>
+        </div>
+    )
+}
+
+const CurrentTicket = () => {
+    const [text, setText] = useState<string>('')
+    const [currentStatus, setCurrentStatus] = useState<string>('')
+    const [pageTime, setPageTime] = useState(0)
+    const [isInternal, setIsInternal] = useState(false)
+    const [isButtonDisabled, setIsButtonDisabled] = useState(false)
+    const [currentDependencies, setCurrentDependencies] = useState<string[]>([])
+
+    const { organizationId } = useOrganization()
+    const { user } = useUserProfile()
+    const uuid = getUuidFromUrl()
+
+    const [mutation] = useChangeStatusMutation()
+    const [mutationTicket] = useSeenTicketCommentMutation()
+    const [mutationCreateComment] = useCreateCommentMutation()
+
+    const { data: ticketData, isLoading: isTicketLoading } = useGetTicketQuery({
+        organization_id: organizationId,
+        ticket_id: uuid[0],
+    })
+
+    const { data: ticketComments, isLoading: isCommentsLoading } = useGetTicketCommentsQuery({
+        ticket_id: uuid[0],
+    })
+
+    const { data, isLoading } = useGetAllEmployeesQuery({
+        organization: organizationId,
+        employee_profile: user.employee_profile?.id,
+    })
+
+    const SendComment = () => {
+        setIsButtonDisabled(true)
+        handleCreateComment(uuid[0], mutationCreateComment, text, data?.results[0].id, isInternal).then((success) => {
+            if (success) {
+                setIsInternal(false)
+                setText('')
+                handleChange
+            }
+            setIsButtonDisabled(false)
+        })
+    }
+
+    const handleChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
+        const { value } = event.target
+        setText(value)
+    }
+
+    const calculateHeight = (text: string): string => {
+        const lineCount = text.split('\n').length
+        const charCount = text.length - text.split('\n').length
+        const extraHeight = Math.floor(charCount / 67) * 22.5
+
+        return `${lineCount * 22.5 + extraHeight}px`
+    }
+
+    const statusTranslations: { [key: string]: { translate: string; color: string } } = {
```

Comment:
Bad naming. It is not translation, it is rather transforming

Reply:
I would locate this dictionary to contstants like it has been done on ticket list page